1.6/community/large/fixtures/jobs.yml

```diff
@@ -1,262 +1,248 @@
 jobs:
     csv_product_import:
-        connector: Akeneo CSV Connector
-        alias:     csv_product_import
-        label:     Demo product import
-        type:      import
+        connector: 'Akeneo CSV Connector'
+        alias: csv_product_import
+        label: 'Product import'
+        type: import
         configuration:
-            filePath:           /tmp/product.csv
-            uploadAllowed:      true
-            delimiter:          ;
-            enclosure:          '"'
-            escape:             '\'
-            enabled:            true
-            categoriesColumn:   categories
-            familyColumn:       family
-            groupsColumn:       groups
+            filePath: /tmp/product.csv
+            uploadAllowed: true
+            delimiter: ;
+            enclosure: '"'
+            escape: \
+            enabled: true
+            categoriesColumn: categories
+            familyColumn: family
+            groupsColumn: groups
             realTimeVersioning: true
-            decimalSeparator: .
-            dateFormat: yyyy-MM-dd
```

Comment:
Please check you can always save your job without this line. I think there is an error in the PIM.